### PR TITLE
Publish ActiveMQ 5.15.11 release

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

**Comment by clebertsuconic** [comment]
@jbonofre did you build.. I don't see any HTML outputs.

You have to edit the .md and call build, and commit within.

if you're having trouble builindg for any environment.. you can always ask for help.

using the view option will also help you.

**Comment by jbonofre** [comment]
@clebertsuconic let me double check (I built AFAIR). Thanks.

**Comment by clebertsuconic** [comment]
if you built, you forgot to add ./content to your build. 

I see no ./content changes, which is what's served by the webservers.

**Comment by clebertsuconic** [comment]
by calling ./build.sh on your PR I see this changed output:

```sh
clebertsuconic$ git status
On branch 22
Your branch is up to date with 'upstream/pr/22'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	content/activemq-51511-release.html
	content/schema/core/activemq-core-5.15.11-schema.html
	content/schema/core/activemq-core-5.15.11-schema.html.asc
	content/schema/core/activemq-core-5.15.11-schema.html.asc.md5
	content/schema/core/activemq-core-5.15.11-schema.html.asc.sha1
	content/schema/core/activemq-core-5.15.11-schema.html.md5
	content/schema/core/activemq-core-5.15.11-schema.html.sha1
	content/schema/core/activemq-core-5.15.11.xsd
	content/schema/core/activemq-core-5.15.11.xsd.asc
	content/schema/core/activemq-core-5.15.11.xsd.asc.md5
	content/schema/core/activemq-core-5.15.11.xsd.asc.sha1
	content/schema/core/activemq-core-5.15.11.xsd.md5
	content/schema/core/activemq-core-5.15.11.xsd.sha1
```

**Comment by jbonofre** [comment]
@clebertsuconic you are right, I didn't commit the output. Let me fix that.

File: src/activemq-51511-release.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+### Git Tag
+
+[https://gitbox.apache.org/repos/asf?p=activemq.git;a=tag;h=refs/tags/activemq-5.15.11](https://gitbox.apache.org/repos/asf?p=activemq.git;a=tag;h=refs/tags/activemq-5.15.11)
+
+Change Log
+----------
+
+For a more detailed view of new features and bug fixes, see the [release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311210&version=12345958)
+
+> **Java 8 Required**
```
Comment (gemmellr) [inline src/activemq-51511-release.md]: Given it has been years, it feels like it is past time this note could go away.

File: src/activemq-51511-release.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+> 
+> The minimum Java version has been upgraded to Java 8.
+
+Also see the previous [ActiveMQ 5.15.9 Release](activemq-5159-release)
```
Comment (gemmellr) [inline src/activemq-51511-release.md]: Links to the wrong previous version.

File: src/schema/core/activemq-core-5.15.11-schema.html.asc.md5 [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<html>
```
Comment (gemmellr) [inline src/schema/core/activemq-core-5.15.11-schema.html.asc.md5]: This file is junk, its not an MD5 file but rather the HTML contents of a 404 error page. It doesn't seem like the MD5 of a signature is needed anyway.

File: src/schema/core/activemq-core-5.15.11-schema.html.asc.sha1 [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<html>
```
Comment (gemmellr) [inline src/schema/core/activemq-core-5.15.11-schema.html.asc.sha1]: This file is junk, its not an SHA1 file but rather the HTML contents of a 404 error page. It doesn't seem like the SHA1 of a signature is needed anyway.

File: src/schema/core/activemq-core-5.15.11.xsd.asc.md5 [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<html>
```
Comment (gemmellr) [inline src/schema/core/activemq-core-5.15.11.xsd.asc.md5]: Ditto re junk content.

File: src/schema/core/activemq-core-5.15.11.xsd.asc.sha1 [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<html>
```
Comment (gemmellr) [inline src/schema/core/activemq-core-5.15.11.xsd.asc.sha1]: Ditto re junk content.

**Comment by jbonofre** [comment]
@gemmellr thanks for your feedback. Fixing the issue (about the file, I checked, it was the same with the previous release as well, fixing).

**Pull request closed without merging**